content/200-orm/800-more/450-migrating-to-prisma/02-migrate-from-sequelize.mdx

```diff
@@ -907,7 +907,7 @@ const createDraft = async (req, res) => {
 }
 ```
 
-Note that Prisma Client's nested write here save an initial query where first the `User` record needs to be retrieved by its `email`. That's because, with Prisma you can connect records in relations using any unique property.
+Note that Prisma Client's nested write here save an initial query where first the `User` record needs to be retrieved by its `email`. That's because, with Prisma ORM you can connect records in relations using any unique property.
```

Comment:
```suggestion
Note that Prisma Client's nested write here save an initial query where first the `User` record needs to be retrieved by its `email`. That's because, with Prisma Client you can connect records in relations using any unique property.
```